src/hourly-weather.ts

```diff
@@ -329,6 +331,7 @@ export class HourlyWeatherCard extends LitElement {
     }
 
     if (!forecastNotAvailable && numSegments > (forecast.length - offset)) {
+      if (pending) return;
```

Comment:
I'm worried about hiding this error in cases where the user has actually tried to configure the card to show more segments than the entity has available. I'm wondering if we should just make `forecastNotAvailable` be true if the forecast is pending.

Reply:
I think this is covered. Am I missing something?
When loading from the attribute (and subscription is available, yet not loaded) the error will be pending and will not be displayed.
Later on when the subscription returns and still number of segments are less than configured pending will be false, thus error is shown.